app/models/manageiq/providers/vmware/infra_manager/template/operations.rb

```diff
@@ -0,0 +1,17 @@
+module ManageIQ::Providers::Vmware::InfraManager::Template::Operations
+  extend ActiveSupport::Concern
+
+  included do
+    supports :terminate do
+      if retired?
+        _('The VM is retired')
+      elsif terminated?
+        _('The VM is terminated')
+      elsif disconnected?
+        _('The VM does not have a valid connection state')
+      elsif !has_active_ems?
+        _("The VM is not connected to an active Provider")
```

Comment:
These are the conditions that are relevant for templates from the `supports :control` block.  Because we were using `supports?(:control)` for templates as well you couldn't terminate a template because it would tell you it was a template.